ecs-logs-collector.sh

```diff
@@ -395,64 +400,99 @@ get_system_services()
   ok
 }
 
-get_docker_info()
+is_docker_healthy()
 {
-  try "gather Docker daemon information"
-
+  try "confirm that Docker is running"
   pgrep docker > /dev/null
   if [[ "$?" -eq 0 ]]; then
-    mkdir -p ${info_system}/docker
-
-    docker info > ${info_system}/docker/docker-info.txt 2>&1
-    docker ps --all --no-trunc > ${info_system}/docker/docker-ps.txt 2>&1
-    docker images > ${info_system}/docker/docker-images.txt 2>&1
-    docker version > ${info_system}/docker/docker-version.txt 2>&1
-
     ok
 
+    if [ `which curl` ]; then
```

Comment:
That wasn't intended. I have now added a warning message if curl is not found, but I still allow collection of information in the usual way since we don't know if the API is working or not.